hivemq-edge/src/main/java/com/hivemq/configuration/entity/EntityValidatable.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2019-present HiveMQ GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hivemq.configuration.entity;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import javax.xml.bind.ValidationEvent;
+import javax.xml.bind.helpers.ValidationEventImpl;
+import java.util.List;
+import java.util.function.BooleanSupplier;
+import java.util.function.Supplier;
+
+public interface EntityValidatable {
+    static boolean notMatch(
```

Comment:
could be default, thus instance methods. they are only used in the context of an instance, internally. or is the plan to use these methods from non implementing instances?. I would find it clearer not to need to read the prefix `EntityValidatable.` in the implementing class.

Reply:
`default` is intrusive, I suppose. Because it may conflict with the same signature from other interface or base class. `static` is not intrusive.